packages/ui-components/src/components/range/range.base.scss

```diff
@@ -3,16 +3,20 @@
 :host {
 	/**
 	 * @prop --range-width: Width of the range slider
+	 * @prop --range-height: Height of the range slider
+	 * @prop --range-selector-width: Width of the range thumb
```

Comment:
`--range-selector-width` and `--range-selector-height` can be merged into one like `--range-selector-radius`